picard/util/__init__.py

```diff
@@ -459,4 +458,20 @@ def convert_to_string(obj):
         return str(obj)
 
 
+def htmlescape(string):
+    return cgi.escape(string)
+
+
+def json_load(data):
+    """
+    Appropriately convert and returns a python dictionary from
+    json string.
+    Args:
+        data (QByteArray): The json response
```

Comment:
Since this uses `convert_to_string`, the input data can at least be of the same types that function supports.